DopplerCustomDomain/CustomDomainProvider/Services.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DopplerCustomDomain.CustomDomainProvider
+{
+    public enum Services
```

Comment:
I would remove this enum, it seems that a simple string would be better to start with.